.github/workflows/0-16-1-auto-merge.yaml

```diff
@@ -0,0 +1,66 @@
+name: Merge pull request into v0.16.1
+
+on:
+  pull_request:
+    types: [closed]
+    branches:
+      - master
+    milestone:
+      types: [created, edited]
+      names:
+        - 'v0.16.1'
+
+jobs:
+  create-pr:
+    runs-on: ubuntu-latest
```

Comment:
I think we need `if: github.event.pull_request.merged == true` here to make sure this isn't triggered when someone closes the PR without merging (see https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#running-your-pull_request-workflow-when-a-pull-request-merges).